test/scenarios/ynEIGEN/ynEigenUpgradeScenarios.t.sol

```diff
@@ -228,33 +286,73 @@ contract ynEigenUpgradeScenarios is ynLSDeScenarioBaseTest {
     }
 
     function upgradeynEigenContracts() internal {
-        TokenStakingNode newTokenStakingNode = new TokenStakingNode();
 
-        TokenStakingNodesManager newTokenStakingNodesManager = new TokenStakingNodesManager();
-        vm.etch(address(tokenStakingNodesManager), address(newTokenStakingNodesManager).code);
+        {
```

Comment:
we shouldn't use etch for our own contracts, replaced all that 

it's not a replica of what happens in production, i'm weary of it

Reply:
@yield-farmer-1  can you spot check me here that this matches your upgrade plan?